reports/jshint-lines-of-shame.js

```diff
@@ -0,0 +1,48 @@
+// TODO: We need to figure out whether React will be supported natively for a reporter or if they
+// will be required to pre-compile
```

Comment:
gotcha. For templates this small I usually just write the react javascript by hand. different strokes is all